taqueria-plugin-taquito/originate.ts

```diff
@@ -49,29 +49,29 @@ const addOrigination = (parsedArgs: Opts, batch: Promise<WalletOperationBatch>)
 		const contractData = await readFile(contractAbspath, 'utf-8');
 		return (await batch).withOrigination({
 			code: contractData,
-			storage: mapping.storage,
+			init: mapping.storage as any,
 		});
 	};
 
 const getValidContracts = async (parsedArgs: Opts) => {
 	const contracts = parsedArgs.contract
 		? [parsedArgs.contract]
-		: (await glob('**/*.tz', { cwd: parsedArgs.config.artifactsDir })) as string[];
+		: (await glob('*.tz', { cwd: parsedArgs.config.artifactsDir })) as string[];
 
 	return contracts.reduce(
-		(retval, filename) => {
-			const storage = getInitialStorage(parsedArgs)(filename);
+		async (retval, filename) => {
+			const storage = await getInitialStorage(parsedArgs, filename);
 			if (storage === undefined || storage === null) {
 				sendErr(
 					`Michelson artifact ${filename} has no initial storage specified for the target environment.\nStorage is expected to be specified in .taq/config.json at JSON path: environment.${
 						getCurrentEnvironment(parsedArgs)
-					}.storage."${filename}"\n`,
+					}.storage["${filename}]\nThe value of the above JSON key should be the name of the file that contains the actual value of the storage, as a Michelson expression.\n`,
```

Comment:
Missing the closing double quote here for the filename

Reply:
Also should this message mention the pathing requirements for the file?

Reply:
Thanks!